x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/users/bulk/upsert_batch.ts

```diff
@@ -35,7 +35,7 @@ export const bulkUpsertBatch =
             {
               '@timestamp': timestamp,
               user: { name: u.username, is_privileged: true },
-              labels: { sources: ['csv'] },
+              labels: { sources: ['csv'], source_ids: [] },
```

Comment:
settings source_ids: [] just brings csv upload in line with the other methods for creating users (and means we can have one assertion utility in our tests 😅 )